CONTRIBUTING.md

```diff
@@ -5,6 +5,13 @@
 If you haven't read it, ***read the very important notice about the Javalib
 in the [Developer documentation](./DEVELOPING.md)***.
 
```

Comment:
W should add here the `import java.{util => ju}` and any other naming convention that we have.